local-volume/README.md

```diff
@@ -64,11 +70,16 @@ $ export KUBE_FEATURE_GATES="PersistentLocalVolumes=true"
 $ export KUBE_FEATURE_GATES="PersistentLocalVolumes=true,MountPropagation=true"
 ```
 
-##### 1.9+
+##### 1.9
 ```
 $ export KUBE_FEATURE_GATES="PersistentLocalVolumes=true,VolumeScheduling=true,MountPropagation=true"
 ```
 
+##### 1.10+
+
+$ export KUBE_FEATURE_GATES="BlockVolumes=true"
```

Comment:
@msau42 - did MountPropogation make it to beta?

Reply:
yup

Reply:
done